src/foremast/templates/pipeline/stage-deploy.json.j2

```diff
@@ -35,6 +35,11 @@
           "suspendedProcesses": [],
           "securityGroups": {{ data.app.instance_security_groups }},
           "tags": {
+            {% if data.app.custom_tags %}
```

Comment:
I think tags is missing the closing bracket and comma, and the loop creates a comma on the last item.  Maybe `tags: {{ data.app.custom_tags | tojson }},` would be easier?

Reply:
@linjmeyer We traditionally always tagged `app_group`, `app_name`, and `owner_email` as you see below. 👍 

We could change it `tojson` however, we might need to refactor to include it in the ec2 code.